projects/core/components/primitive-calendar/primitive-calendar.component.ts

```diff
@@ -65,10 +65,7 @@ export class TuiPrimitiveCalendarComponent {
 
     @HostBinding('class._single')
     get isSingle(): boolean {
-        return (
-            this.value !== null &&
-            (this.value instanceof TuiDay || this.value.isSingleDay)
-        );
+        return this.value instanceof TuiDayRange && this.value.isSingleDay;
```

Comment:
It is difficult to understand why `isSingle` should return `false` for `TuiDay`.

Should we rename getter to more descriptive ? 🤔 
Something like `rangeWithSingleDay` ?

Reply:
`isSingleDayRange` sounds better to my ear :)

Reply:
fixed